runtime/doc/eval.txt

```diff
@@ -4053,6 +4053,8 @@ jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
 		  - width: Width of the terminal screen(only if pty is set)
 		  - height: Height of the terminal screen(only if pty is set)
 		  - TERM: $TERM environment variable(only if pty is set)
+		  - ignore_stdin: If set, the job will not open an stdin
+		    channel (and jobsend() won't work).
```

Comment:
Taking a hint [from nodejs](https://github.com/neovim/neovim/issues/2757#issue-82149016) should we anticipate a request for similar features for stdout, stderr? Rather than adding more booleans perhaps we can adapt the nodejs:

    { stdio: ['ignore', 'pipe', 'pipe'] })